### PR TITLE
[Issue #356] Moved specific script parts to related environments

### PR DESCRIPTION
**Pull request by yannlabou**
relates to #403 and #356

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -17,11 +17,31 @@ matrix:
     - os: linux
       env: CUDA=8.0.61-1 CMAKE_OPTIONS="-DPENGUINV_BUILD_TEST=ON"
       before_install: source script/travis/install_cuda.sh
+      script:
+        - |
+          if [[ $CUDA ]]; then
```
Comment (ihhub) [inline .travis.yml]: I think we can remove this condition as we know that this job is for CUDA only.

**Comment by ihhub** [comment]
@yannlabou I made small corrections in your pull request. Could you please take a look into them and let me know your opinion? :)

**Comment by yannlabou** [comment]
You changed only the redundant CUDA env check. Looks good to me.

**Comment by ihhub** [comment]
@yannlabou, great work as usual!

**Pull request merged**